docker-compose.yaml

```diff
@@ -465,6 +465,7 @@ services:
       - DB_NAME=zac
       - DB_PASSWORD=password
       - DB_USER=zac
+      - FEATURE_FLAG_BPMN_SUPPORT=${FEATURE_FLAG_BPMN_SUPPORT:-false}
```

Comment:
Shouldn't this be true for local environment?

Reply:
no

Reply:
At least I don't think so. It's spike functionality anyway and there is not really a reason to enabled it anywhere by default for now, not even locally.